.travis.yml

```diff
@@ -15,16 +15,19 @@ env:
       - TARGET=test VERSION=1_7
       - TARGET=test VERSION=1_9
       - TARGET=test VERSION=1_10
-      - TARGET=test VERSION=2_0
+      - TARGET=test VERSION=2x
+      - TARGET=test VERSION=2_2
       - OS=el DIST=6
       - OS=el DIST=7
       - OS=fedora DIST=26
       - OS=fedora DIST=27
+      - OS=fedora DIST=28
+      - OS=fedora DIST=29
       - OS=ubuntu DIST=trusty
       - OS=ubuntu DIST=xenial
       - OS=ubuntu DIST=artful
       - OS=ubuntu DIST=bionic
-      - OS=debian DIST=wheezy
+      - OS=ubuntu DIST=cosmic
       - OS=debian DIST=jessie
       - OS=debian DIST=stretch
 
```

Comment:
```suggestion
      - OS=debian DIST=buster
      - OS=debian DIST=sid

```

Reply:
At least we need tarantool on sid in our repository (and appropriate testing pipelines) to use it as a dependency in a module. So I would expect until https://github.com/tarantool/tarantool/issues/4037 will be resolved or declined and only then going to update modules.

Reply:
I see, let's wait then :)